src/chrome/crdpMultiplexing/webSocketToLikeSocketProxy.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { logger } from 'vscode-debugadapter';
+import * as WebSocket from 'ws';
+import { LikeSocket } from 'noice-json-rpc';
+
+export class WebSocketToLikeSocketProxy {
+    private _server: WebSocket.Server;
+
+    constructor(private _port: number, private _socket: LikeSocket) { }
+
+    public start(): void {
+        this._server = new WebSocket.Server({ port: this._port }, () => {
+            logger.log(`CRDP Proxy listening on: ${this._port}`);
+        });
+
+        this._server.on('connection', openedWebSocket => {
+            logger.log(`CRDP Proxy accepted a new connection`);
+            openedWebSocket.on('message', data => {
```

Comment:
I guess I'm not sure why you have this socket receiving messages and forwarding them to another socket, which is just open in the same process. ie, why is the Channel a LikeSocket?